EasyPost/Http/ClientConfiguration.cs

```diff
@@ -120,11 +116,6 @@ private abstract class Defaults
             ///     The default connection timeout in milliseconds.
             /// </summary>
             internal const int DefaultConnectTimeoutMilliseconds = 30000;
```

Comment:
issue: we want the timeout to be 60 seconds when only one of the two can be set. Please adjust accordingly.

Additionally, when only one is available, it's usually the request timeout but here we are retaining the connection timeout, can we determine for certain which one the .NET http package actually exposes and rename if necessary?

Reply:
The Http client simply has a property called `Timeout`. We can rename it to that to make it simple.

<img width="1019" alt="image" src="https://user-images.githubusercontent.com/17054780/234366151-53001cd1-793a-40ac-b4de-8eab74874782.png">